front/lib/email.ts

```diff
@@ -157,3 +157,35 @@ export async function sendAdminSubscriptionPaymentFailedEmail(
   };
   return sendEmail(email, message);
 }
+
+export async function sendAdminDataDeletionEmail({
+  email,
+  firstName,
+  workspaceName,
+  remainingDays,
+  isLast,
+}: {
+  email: string;
+  firstName: string;
+  workspaceName: string;
+  remainingDays: number;
+  isLast: boolean;
+}): Promise<void> {
+  const msg = {
+    from: {
+      name: "Dust team",
+      email: "team@dust.tt",
+    },
+    subject: `${
```

Comment:
Let's add [Dust] ?

Reply:
Sure -- It was there at first but from the slack thread I was under the impression @Duncid  wanted to remove it